accessible_api/AccessibilityAPIServer.py

```diff
@@ -0,0 +1,42 @@
+""" API for Accessible Requests """
+
+from SocketServer import TCPServer
+from argparse import ArgumentParser
+from AccessibilityRequestHandler import PlatformAccessibilityRequestHandler
+
+
+VERBOSE = False
+
+
+class AccessibilityAPIServer:
+    def __init__(self, port, platform=None, ip=""):
+        if VERBOSE:
+            print '.............SETTING UP SERVICE............'
+        handler = PlatformAccessibilityRequestHandler(platform)
```

Comment:
pylint is showing the invalid function name error when i use platform_accessibility_request_handler_factory